build_function_part_prices.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash/
+pip3 install --target ./package sqlalchemy
+pip3 install --target ./package pymysql
+pip3 install --target ./package requests-oauthlib
```

Comment:
```suggestion
pip3 install --target ./package sqlalchemy pymysql requests-oauthlib
```

or use a requirements.txt